app/assets/stylesheets/_settings.scss

```diff
@@ -618,7 +618,7 @@ $tooltip-radius: $global-radius;
 $topbar-padding: 0.5rem;
 $topbar-background: $light-gray;
 $topbar-submenu-background: $topbar-background;
-$topbar-title-spacing: 0.5rem 1rem 0.5rem 0;
+$topbar-title-spacing: 0 2rem 0 0;
```

Comment:
This should be in a separate issue, but I'm also not sure why we'd want to drop the spacing to zero.